website/src/components/Changelog/GUI.tsx

```diff
@@ -8,7 +8,13 @@ export default function GUI({ os }: { os: OS }) {
   return (
     <Entries downloadLinks={downloadLinks(os)} title={title(os)}>
       {/* When you cut a release, remove any solved issues from the "known issues" lists over in `client-apps`. This must not be done when the issue's PR merges. */}
-      <Unreleased></Unreleased>
+      <Unreleased>
+        {os === OS.Windows && (
+          <ChangeItem pull="9203">
+            Allows managing certain settings via Microsoft Intune.
```

Comment:
Is this specific to Intune? They are available to any MDM right?

Reply:
If the MDM can set registry keys, yes.

Reply:
Yeah I think these templates _should_ apply to any MDM, so probably fine to just say this is general managed config support or something.